consensus/polybft/consensus_runtime.go

```diff
@@ -490,66 +496,105 @@ func (c *consensusRuntime) restartEpoch(header *types.Header) (*epochMetadata, e
 	}, nil
 }
 
-// calculateCommitEpochInput calculates commit epoch input data for blocks starting from the last built block
-// in the current epoch, and ending at the last block of previous epoch
-func (c *consensusRuntime) calculateCommitEpochInput(
-	currentBlock *types.Header,
-	epoch *epochMetadata,
-) (*contractsapi.CommitEpochValidatorSetFn,
-	*contractsapi.DistributeRewardForRewardPoolFn, error) {
-	uptimeCounter := map[types.Address]int64{}
-	blockHeader := currentBlock
-	epochID := epoch.Number
-	totalBlocks := int64(0)
+// calculateCommitEpochInput calculates commit epoch input data
+func calculateCommitEpochInput(
```

Comment:
Maybe we should rename this helper function to something like `getCommitEpochInput` or similar and make that function outside of `consensusRuntime`. There is no calculation, so name is misleading